DOCS-58 - Anomaly detection design updates (#3853)
SumoLogic/sumologic-documentation: docs/alerts/monitors/use-playbooks-with-monitors.md

Weekly seasonality detection is turned off by default to optimize performance. [Contact Sumo Logic Customer Support](https://support.sumologic.com/support/s/contactsupport) to activate it for specific monitors. (*Weekly seasonality detection* is the optimization of baseline calculations to account for the variations of data flow that can occur in a work week.)
:::

> Watch the following video to learn about anomaly monitors.

Reviewer: ```suggestion
Watch this micro lesson to learn about anomaly monitors.
```
